index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Sodiq's Portfolio</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/eb94992e2c.js" crossorigin="anonymous"></script>
+</head>
+<body>
+  <div class="wrapper">
+    <nav class="top-nav">
+      <ul>
+        <li class="time">9:19 am</li>
+        <li class="top-icons">
+          <span><i class="fa-solid fa-signal"></i></span>
+          <span><i class="fa-solid fa-wifi"></i></span>
+          <span><i class="fa-solid fa-battery-full"></i></span>
+        </li>
+      </ul>
+    </nav>
+    <nav class="menu-nav">
+      <ul>
+        <li class="name">Sodiq A.</li>
+        <li class="menu-icon"><i class="fa-solid fa-bars"></i></li>
+      </ul>
+    </nav>
+    
+    <header class="header">
+      <div class="intro">
+        <h1>Hey there. I'm Sodiq. <br> <span>I'm a Software Developer</span></h1>
+        <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded,  don’t hestiate to contact me.</p>
+      </div>
+      
+      <div class="socials">
+        <ul>
+          <li>
+            <span><a href="#"><i class="fa-brands fa-github"></i></a></span>
+            <span><a href="#"><i class="fa-brands fa-linkedin"></i></a></span>
+            <span><a href=""><i class="fa-brands fa-angellist"></i></a></span>
+            <span><a href="#"><i class="fa-brands fa-twitter"></i></a></span>
+            <span><a href="#"><i class="fa-brands fa-medium"></i></a></span>
+          </li>
+        </ul>   
+      </div>
+      
+      <div class="arrow">
+        <i class="fa-solid fa-circle-chevron-down"></i>
+      </div>
+    </header>
```

Comment:
- [ ] Kindly wrap this inside a `section` tag
- [ ] Please wrap each icon in a `li` tag

Reply:
Thanks for the second review. My initial code was all in <li> tags but I ran into issues while I was applying flexbox to them. This review has forced me to figure out the right way to do it and I learned something new with it.